src/conf_mode/interfaces-ethernet.py

```diff
@@ -77,6 +77,8 @@ def verify(ethernet):
 
     # verify offloading capabilities
     if 'offload' in ethernet and 'rps' in ethernet['offload']:
+        if os.cpu_count() < 2:
+            raise ConfigError('RPS cannot be enabled on a system with only 1 logical CPU!')
```

Comment:
This is incorrect - works on VMware with only one vCPU. The problem must be of a different kind.

Reply:
@c-po I have a question, is it possible to judge only for KVM?

Reply:
done